tests/docker/BUILD

```diff
@@ -39,3 +39,53 @@ container_test(
     configs = ["//tests/docker/configs:deb_image_with_dpkgs.yaml"],
     image = ":deb_image_with_dpkgs",
 )
+
+container_image(
+    name = "image_with_some_cmd_entrypoint",
```

Comment:
set_cmd_and_entrypoint

also add tests with just cmd and just entrypoint

Reply:
Done.